packages/ketrew/ketrew.3.0.0/opam

```diff
@@ -0,0 +1,36 @@
+opam-version: "1.2"
+maintainer: "seb@mondet.org"
+authors : [
+  "Sebastien Mondet <seb@mondet.org>"
+  "Leonid Rozenberg <leonidr@gmail.com>"
+  "Arun Ahuja <aahuja11@gmail.com>"
+  "Jeff Hammerbacher <jeff.hammerbacher@gmail.com>"
+  "Isaac Hodes <isaachodes@gmail.com>"
+]
+homepage: "http://www.hammerlab.org/docs/ketrew/master/index.html"
+dev-repo: "https://github.com/hammerlab/ketrew.git"
+bug-reports: "https://github.com/hammerlab/ketrew/issues"
+available : [ ocaml-version >= "4.02.2" ]
+install: [
+  ["omake"]
+  ["omake" "install" "BINDIR=%{bin}%"]
+]
+remove: [
+  ["ocamlfind" "remove" "ketrew"]
+  ["ocamlfind" "remove" "ketrew_pure"]
+  ["rm" "-f" "%{bin}%/ketrew"]
+]
+depends: [
+  "omake" "ocamlfind" "ocamlify"
```

Comment:
It would be good to annotate the build-only deps with `{build}` in the future. :-)